samples/contrib/aws-samples/README.md

```diff
@@ -0,0 +1,198 @@
+# Sample AWS SageMaker Kubeflow Pipelines 
+
+This folder contains many example pipelines which use [AWS SageMaker Components for KFP](https://github.com/kubeflow/pipelines/tree/master/components/aws/sagemaker). Following sections explain the setup you need to run these pipelines. Once you are done with the setup, [simple_train_pipeline](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/aws-samples/simple_train_pipeline) is a good place to start if you have never used these components before.
+
+
+
+## Prerequisites 
+
+1. You need a cluster with Kubeflow installed on it. [Install Kubeflow on AWS cluster](https://www.kubeflow.org/docs/aws/deploy/install-kubeflow/)
```

Comment:
Specifically (for EKS OIDC) they need an EKS cluster, not just any K8s.

Reply:
Added a line in IAM permissions section 
stating that OIDC requires EKS cluster